src/test/java/io/github/oliviercailloux/collaborative_exams/helper/QuestionTextTest.java

```diff
@@ -17,8 +18,10 @@
 
 	@Deployment(testable = false)
 	public static WebArchive createDeployment() {
-		final WebArchive war = ShrinkWrap.create(WebArchive.class, "collaborative-exams-0.0.1.war")
-				.addPackage(QuestionTextTest.class.getPackage());
+		final WebArchive war = ShrinkWrap.create(WebArchive.class, "examcollab.war")
+				.addPackage(QuestionTextTest.class.getPackage())
+				.addPackages(true, "org.apache.deltaspike")
+				.addAsManifestResource(EmptyAsset.INSTANCE, "META-INF/beans.xml");
```

Comment:
Pourquoi utiliser beans.xml, censé être inutile ? S’il est absolument nécessaire, en documenter la raison, de préférence avec un lien vers une doc officielle et récente (éventuellement https://www.ibm.com/support/knowledgecenter/en/SS7JFU_8.5.5/com.ibm.websphere.express.doc/ae/cweb_cdi.html , bien que non idéal, documente une ancienne spec, CDI 1.0).

Reply:
Il ne m'est actuellement pas possible d'effectuer des tests et ce depuis une dizaine d'heures. Les jobs Travis restent bloqués sur "queued".

beans.xml semble nécessaire si on en croit le message d'erreur "**Could not find beans** for Type=class org.apache.deltaspike.core.impl.scope.window.WindowBeanHolder and qualifiers:[]"


Egalement si on en croit les posts suivants :
https://developer.jboss.org/thread/237174
http://abulletproofidea.com/questions/50927/test-cdi-avec-arquillian
https://developer.jboss.org/thread/237358
https://stackoverflow.com/questions/33169361/arquillian-configuration-deploymentexception-with-windowbeanholder
http://apache-deltaspike-incubator-discussions.2316169.n4.nabble.com/Wled-2-1-2-Could-not-find-beans-for-Type-class-org-apache-deltaspike-core-impl-scope-window-WindowBe-td4656801.html
https://users.openejb.apache.narkive.com/5cYxHyxG/applicationcomposer-and-deltaspike-1-0